master/internal/db/postgres_experiments.go

```diff
@@ -1132,9 +1132,9 @@ func (db *PgDB) AddExperiment(experiment *model.Experiment) (err error) {
 		}
 		err := namedGet(tx, &experiment.ID, `
 	INSERT INTO experiments
-	(state, config, model_definition, start_time, end_time, archived, parent_id,
+	(state, config, model_definition, start_time, end_time, archived, parent_id, progress,
```

Comment:
Not your PR's issue, but it's really strange progress isn't just in the model to me.